sdk/python/jobs/pipelines/1e_pipeline_with_registered_components/Model Import Pipeline.ipynb

```diff
@@ -0,0 +1,287 @@
+{
```

Comment:
Please run spell check. There are a few typos.

Reply:
Done, Thank you :)